apps/prairielearn/assets/scripts/lib/histogram.ts

```diff
@@ -133,9 +121,5 @@ export function histogram(
     .attr('x', -height / 2)
     .attr('y', '-3em')
     .style('text-anchor', 'middle')
-    .text(options.ylabel);
-
-  svg.append('line').attr({ x1: 0, y1: 0, x2: width, y2: 0, class: 'x axis' });
-
-  svg.append('line').attr({ x1: width, y1: 0, x2: width, y2: height, class: 'y axis' });
```

Comment:
Were these removed by mistake?

Reply:
No, these were removed intentionally as they were not actually rendering when written like this. `.attr()` only accepts two arguments as strings.  While working with @nwalters512 , we tested this using `attr()` for setting each attribute and decided that these were bold axis lines that were not improving the appearance of the histogram and best to remove.